### DISCUSSION TITLE
Is it possible to map "Ctrl-scrollwheel" for changing text size?

### DISCUSSION THREAD

**Discussion by precompute** [kovidgoyal/kitty#5069] [Q&A] [answered]
Moving from xfce4-terminal, and missing this feature.  Is there any way I could map `C-scroll wheel up` to zoom out and `down` to zoom in?

**Comment by kovidgoyal** [comment, marked as answer]
No, use ctrl+plus or ctrl+minus instead

On Sun, May 08, 2022 at 02:34:41AM -0700, Terminal wrote:
> Moving from xfce4-terminal, and missing this feature.  Is there any way I could map `C-scroll wheel up` to zoom out and `down` to zoom in?
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/5069
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by precompute** [reply]
Thanks!
